Makefile

```diff
@@ -116,24 +125,52 @@ defn: $(defn_files)
 
 # OCAML Backend
 
-.build/ocaml/driver-kompiled/interpreter: $(ocaml_files) KRYPTO.ml
+ifeq ($(BYTE),yes)
+EXT=cmo
+LIBEXT=cma
+DLLEXT=cma
+OCAMLC=c
+LIBFLAG=-a
+else
+EXT=cmx
+LIBEXT=cmxa
+DLLEXT=cmxs
+OCAMLC=opt -O3
+LIBFLAG=-shared
+endif
```

Comment:
Can this be indented (perhaps with spaces if not tabs)? Looks strange.

Reply:
fixed